src/main/fc/rc.c

```diff
@@ -106,13 +107,18 @@ bool getShouldUpdateFeedforward()
 }
 
 float getSetpointRate(int axis)
-// only used in pid.c to provide setpointRate for the crash recovery function
 {
+#ifdef USE_RC_SMOOTHING_FILTER
     return setpointRate[axis];
+#endif
+    return rawSetpoint[axis];
 }
 
 float getRcDeflection(int axis)
 {
+#ifdef USE_RC_SMOOTHING_FILTER
+    return rcDeflectionSmoothed[axis];
+#endif
```

Comment:
For readability's sake I'd put the second return into an `#else` conditional.

Reply:
Done